test-coverage.txt

```diff
@@ -0,0 +1,43 @@
+ PASS  src/components/Header/DarkmodeButton/DarkmodeButton.test.js
+ PASS  src/components/Header/ProfileButton/ProfileButton.test.js
+ PASS  src/components/Header/MenuButton/MenuButton.test.js
+ PASS  src/components/Header/Searchbar/Searchbar.test.js
+ PASS  src/components/VideosGrid/VideoGridItem/VideoGridItem.test.js
+ PASS  src/components/Header/Header.test.js
+ PASS  src/components/VideosGrid/VideoGrid.test.js
+ PASS  src/components/Layout/Layout.test.js
+ PASS  src/components/App/App.test.js
+-------------------------------------|----------|----------|----------|----------|-------------------|
+File                                 |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+-------------------------------------|----------|----------|----------|----------|-------------------|
+All files                            |    93.18 |       50 |    92.31 |    93.18 |                   |
```

Comment:
Here you have a high percentage on everything except the branch column, you can try to solve this by creating more cases, most of the time this percentage is low because the events in some components are untested, you can look at the files generated by jest and search for the untested code